content/en/docs/concepts/cluster-administration/system-metrics.md

```diff
@@ -98,6 +98,14 @@ Take metric `A` as an example, here assumed that `A` is deprecated in 1.n. Accor
 
 If you're upgrading from release `1.12` to `1.13`, but still depend on a metric `A` deprecated in `1.12`, you should set hidden metrics via command line: `--show-hidden-metrics=1.12` and remember to remove this metric dependency before upgrading to `1.14`
 
+## Disable Accelerator Metrics
+
+Since Kubernetes release 1.9, the kubelet has been collecting Accelerator metrics through cadvisor. To collect such metrics, for accelerators such as NVIDIA GPUs, Kubelet held an open handle on that driver. This meant that to perform infrastructure changes (e.g: Update the driver), a cluster administrator needed to stop the kubelet agent.
```

Comment:
minor:

```suggestion
Since Kubernetes release 1.9, the kubelet has been collecting Accelerator metrics through cAdvisor. To collect these metrics for accelerators such as NVIDIA GPUs, kubelet held an open handle on that driver. This meant that to perform infrastructure changes (e.g. update the driver), a cluster administrator needed to stop the kubelet agent.
```

Reply:
Fixed, thanks!

Reply:
Fixed, thanks!